overlord/snapstate/handlers.go

```diff
@@ -1600,6 +1615,16 @@ func (m *SnapManager) undoLinkSnap(t *state.Task, _ *tomb.Tomb) error {
 		return err
 	}
 
+	if newInfo.GetType() == snap.TypeSnapd {
+		// only way to get
+		deviceCtx, err := DeviceCtx(st, t, nil)
+		if err != nil {
+			return err
+		}
+		const noReboot = false
```

Comment:
I really like that you used a const here to give the bool meaning :) ! It's a double negative (and slightly confusing), could we write this as a positive maybe? `const rebootRequired = false` ?